en/core-libraries/form.rst

```diff
@@ -98,6 +98,42 @@ accordingly. We could have also used the ``validate()`` method to only validate
 the request data::
 
     $isValid = $form->validate($this->request->data);
+    
+Setting Form Values
+===================
+
+In order to set the values for the fields of a modelless form, you can define
```

Comment:
Since this section deals with modelless forms, I think that sentence still is correct and not misleading?